yin_yang/helpers.py

```diff
@@ -0,0 +1,50 @@
+import subprocess
+
+
+"""Check output of a command.
+
+This is a helper method which will change how we check output depending on if
+The application is running in a Flatpak or not.
+"""
+
+
+def check_output(args, universal_newlines=False) -> bytes:
+    try:
+        output = subprocess.check_output(
+            args=args, universal_newlines=universal_newlines
+        )
+        return output
+    except FileNotFoundError:
+        flatpak_args = ["flatpak-spawn", "--host"] + args
```

Comment:
This is used several times, so maybe make these args a global variable in this file, or create a new function. Makes editing less annoying, if we need to do that at some point.

Reply:
Moved to global variable.